README.md

```diff
@@ -0,0 +1,20 @@
+[![Coverage Status](https://coveralls.io/repos/github/praekelt/seed-auth-api/badge.svg?branch=develop)](https://coveralls.io/github/praekelt/seed-auth-api?branch=develop)
+[![Build Status](https://travis-ci.org/praekelt/seed-auth-api.svg?branch=develop)](https://travis-ci.org/praekelt/seed-auth-api)
+[![Requirements Status](https://requires.io/github/praekelt/seed-auth-api/requirements.svg?branch=feature%2Fissue-2-create-django-skeleton)](https://requires.io/github/praekelt/seed-auth-api/requirements/?branch=feature%2Fissue-2-create-django-skeleton)
+
+# seed-auth-api
+Seed Auth API. User and permissions store, authentication and authorization.
+
+## Running
+
+ * `pip install -e .`
+ * `psql -U postgres -c "CREATE DATABASE seed_auth;`
+ * `./manage.py migrate`
+ * `./manage.py createsuperuser`
+ * `./manage.py runserver --settings=seed_auth_api.testsettings`
```

Comment:
Wondering whether our dev environment settings would ever differ from test settings (in which case maybe we need a dev settings module).